usr/bin/whonix-gateway-firewall

```diff
@@ -165,6 +165,8 @@ variables_defaults() {
     if test -f /usr/share/qubes/marker-vm; then
       LOCAL_NET_IP6="\
             ::1/128 \
+            fd09:24ef:4179::a8a:/112 \
+            fd09:24ef:4179::a89:/112 \
```

Comment:
Just so these can be double-checked, where did these particular IPs come from in Qubes OS?

Reply:
These were IP address spaces when creating VMs. I couldn't find any specific documentation on them, but there is also an open pull request which references the same networks. https://github.com/QubesOS/qubes-core-admin/pull/558